### PR TITLE
Updates for release 10 2 3

### PR DESCRIPTION
**Pull request by popescu-v**
Updates for the 10.2.3 release.

### CODE REVIEW

File: docs/setup/index.md
Comment (lucaurelien) [inline docs/setup/index.md:1]: To align with docs/setup.conda.md, the conda command for Windows should include `-c conda-forge`.

For macOS, I suggest very simple update for line 27:
```
 - macOS 12 or later, via :simple-anaconda: **Conda** only. Full support for ARM, limited support for Intel.
```

Comment (popescu-v) [reply docs/setup/index.md:1]: Yes, good catch! I will will update this.

For the other point, IMHO it is not clear what does "full" versus "limited" means.

Comment (popescu-v) [reply docs/setup/index.md:1]: Done & Done.

**Review by lucaurelien — APPROVED** [review]
LGTM

**Pull request merged**